draft-ietf-webtrans-http3.md

```diff
@@ -213,9 +213,12 @@ support, both the client and the server MUST send a max_datagram_frame_size
 transport parameter with a value greater than 0 (see {{Section 3
 of !QUIC-DATAGRAM=RFC9221}}).
 
+Any WebTransport requests sent by the client without enabling QUIC and HTTP
+datagrams MUST be treated as malformed by the server.
```

Comment:
Does this mean the server should send a 400 or a H3_MESSAGE_ERROR?

Reply:
Here's what 9114 says:

> Malformed requests or responses that are detected MUST be treated as a [stream error](https://datatracker.ietf.org/doc/html/rfc9114#errors) of type [H3_MESSAGE_ERROR](https://datatracker.ietf.org/doc/html/rfc9114#H3_MESSAGE_ERROR).

> For malformed requests, a server MAY send an HTTP response indicating the error prior to closing or resetting the stream.

So MUST reset and MAY send a response first (hope it's a reliable reset!).  I like this since it handles the case where the server might have already assumed the settings would be correct, and sent a 200, and detected malformed-ness later.

Reply:
I like this.  Could you add a direct citation into the spec text?

Reply:
```suggestion
datagrams MUST be treated as malformed by the server,
as defined in {{Section 4.1.2 of HTTP3}}.
```